README.md

```diff
@@ -244,6 +244,20 @@ Currently, `ctkey-wrapper` requires the user to be running the openconnect-tinyp
 container [here](https://github.com/trussworks/openconnect-tinyproxy) to connect
 to Cloudtamer.
 
+### Verify serverless setup
+
+To verify serverless (and AWS access) is set up properly with ctkey, run:
+
+ ```shell
+which serverless # should return something like /managed-care-review/scripts/serverless`
+which sls # should return something like /managed-care-review/scripts/sls`
+```
+
+These should both point to paths inside the codebase (not to paths in usr/local/bin).
+
```

Comment:
Cool, looks great, thanks for adding this!

Just a typo on this path, '/usr/local/bin'.